pkg/server/helpers.go

```diff
@@ -404,3 +405,20 @@ func (c *criContainerdService) ensureImageExists(ctx context.Context, ref string
 	}
 	return &newImage, nil
 }
+
+// resolveSymbolicLink resolves a possbile symlink path. If the path is a symlink, returns resolved
+// path and true; if the path is not a symlink, returns original path and false.
+func resolveSymbolicLink(path string) (string, bool, error) {
+	info, err := os.Lstat(path)
+	if err != nil {
+		return "", false, err
+	}
+	if info.Mode()&os.ModeSymlink != os.ModeSymlink {
+		return path, false, nil
+	}
+	resolved, err := filepath.EvalSymlinks(path)
```

Comment:
I read it as keeps relative unless the symlink resolves to an absolute.. thus my question? I wasn't sure if it needed to stay relative. One would hope it doesn't matter.